### ISSUE TITLE
Java 17 enhancement support - Unsupported class file major version 61

### ISSUE THREAD

**Issue by rbygrave** [ebean-orm/ebean#2239]
```
10:08:45.747 [main] INFO  io.ebean.EbeanVersion - ebean version: 12.9.0
ebean-enhance> cls: io/avaje/config/InitialLoadContext  msg: defaulted common supertype for type1:java/io/InputStream type2:java/io/FileInputStream due to err:java.lang.IllegalArgumentException: Unsupported class file major version 61
ebean-enhance> cls: io/avaje/config/InitialLoadContext  msg: defaulted common supertype for type1:java/io/InputStream type2:java/lang/Object due to err:java.lang.IllegalArgumentException: Unsupported class file major version 61
ebean-enhance> cls: io/avaje/config/InitialLoadContext  msg: defaulted common supertype for type1:java/io/InputStream type2:java/io/FileInputStream due to err:java.lang.IllegalArgumentException: Unsupported class file major version 61
ebean-enhance> cls: io/avaje/config/InitialLoadContext  msg: defaulted common supertype for type1:java/io/InputStream type2:java/lang/Object due to err:java.lang.IllegalArgumentException: Unsupported class file major version 61
```

**Comment by rbygrave** [comment]
Fixed with ebean-agent 12.9.1 via https://github.com/ebean-orm/ebean-agent/pull/162

**Comment by rbygrave** [comment]
ebean-agent 12.9.1 has been released and an updated IntelliJ plugin submitted for approval.
